metagraph-plugin/conda_recipe/meta.yaml

```diff
@@ -29,6 +32,14 @@ test:
   imports:
     - metagraph
     - katana
+    - pytest
```

Comment:
NIT: These are things you want it to *test* by importing. It would be better to call them "import tests" not "imports". So the things you have listed here (which I think are your dependencies), should not be, but it would be reasonable to have `metagraph_katana` listed here since it's the python package that is install in this conda package and which we are testing here.

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-imports

Reply:
Since the entry point of metagraph_katana is not "import metagraph_katana" but something part of "metagraph", should we add something like "metagraph.wrappers.Graph.KatanaGraph" to the import tests? 

Besides, this is for the "type" plugin of katana only. For our "algorithm" and "translator" plugin, they are written in a decorator manner, and I am not sure how to import them. (I could find them somewhere I think, but the import test might looks ugly)

Reply:
You don't need to have an `imports` section. It's just a quick way to test code by importing it. Your pytests will be better and more complete anyway. You can just remove this.

Reply:
Yeah, It is redundant. Agree.